frontend/src/metabase/admin/upsells/UpsellHosting.tsx

```diff
@@ -44,6 +45,7 @@ export const UpsellHostingUpdates = ({ source }: { source: string }) => {
       buttonLink="https://www.metabase.com/cloud"
       illustrationSrc={RocketGlobeIllustrationSrc}
       source={source}
+      maxWidth={202}
```

Comment:
As explained in the above comment, it used to wrap after I changed `box-sizing`
![Screenshot 2024-07-11 at 8 10 11 PM](https://github.com/metabase/metabase/assets/1937582/7e1ae3fa-e4ca-41e5-b73f-24a6af7e7bba)